docker/build.gradle

```diff
@@ -0,0 +1,62 @@
+task wrapper(type: Wrapper) {
+    gradleVersion = '2.9'
+}
+
+
+buildscript {
+    repositories {
+        jcenter()
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath 'com.bmuschko:gradle-docker-plugin:2.6.1'
+    }
+}
+
+repositories {
+    mavenCentral()
+}
+
+apply plugin: 'com.bmuschko.docker-remote-api'
+
+import com.bmuschko.gradle.docker.tasks.image.Dockerfile
+import com.bmuschko.gradle.docker.tasks.image.DockerBuildImage
+
+task copyExecutorJar(type:Copy){
+    from '../myriad-executor/build/libs/myriad-executor-runnable-0.1.0.jar'
```

Comment:
As Darin suggested in other places, this seems deprecated and no need to copy this.